index.html

```diff
@@ -7,11 +7,17 @@
     <meta name="viewport" content="width=device-width,initial-scale=1" />
     <title>my Portfolio</title>
     <link rel="stylesheet" href="style/style.css" />
+    <link rel="stylesheet" href="style/desktop-style.css" />
   </head>
 
   <body>
     <header class="header">
-      <a href="#" class="my-logo"> Milen </a>
+      <a href="#" class="my-logo"> <h3>Milen</h3> </a>
+      <ul>
+        <li><a href="#">Portfolio </a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#"> Contact</a></li>
+      </ul>
       <nav>
         <div class="my-button">
           <img alt="nav" class="menu" src="./images/Enabled.png" />
```

Comment:
- [ ] Make sure to improve the header styling to make it stick to the Figma design. I think you can check on the **spacing** at right and left, **color** and **spacing** of the **nav links** (portfolio, about, contact,...), and the **Fonts**.
#### Figma design
![st_f](https://user-images.githubusercontent.com/83042742/173113581-be1888fe-eb50-4091-919c-5139a575df3d.png)
#### Your design
![st_h](https://user-images.githubusercontent.com/83042742/173113633-0a145878-17d8-49bc-b1b5-79b315ac5237.png)